.gitlab/ci/global.yml

```diff
@@ -39,6 +39,14 @@
   - ./Tests/scripts/open_ssh_tunnel.sh
   - section_end "Open SSH Tunnel"
 
+.check_build_files_are_up_to_date: &check_build_files_are_up_to_date
+  - section_start "Check Build Files Are Up To Date"
+  - ./Tests/scripts/is_file_up_to_date.sh .gitlab $CI_COMMIT_BRANCH
```

Comment:
add a condition that we are not checking it on nightly (content & SDK) and bucket uploads.
this will avoid problems when someone is pushing something new after the build started.